src/Application/ConfigResolver.php

```diff
@@ -39,7 +39,7 @@ public static function resolve(array $config, string $directory): array
 
         foreach (self::$presets as $presetClass) {
             if ($presetClass::getName() === $preset) {
-                $config = array_merge_recursive($presetClass::get(), $config);
+                $config = array_replace_recursive($presetClass::get(), $config);
```

Comment:
It's possible to add a test on this?

Reply:
If you are not in that mood, I will merge it anyway.

Reply:
Will do. I was looking into adding a test. Give me some time.

Reply:
If you have time, you can contribute later with tests. Thanks!